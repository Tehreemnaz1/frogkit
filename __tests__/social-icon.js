import { mount } from 'vue-test-utils'
import Vue from 'vue'
import SocialIcon from '@/components/social-icon'

const socialItems = {
  headline: 'Smartfrog folgen:',
  icons: [
    {
      name: 'facebook',
      url: 'http://www.facebook.com/smartfrogDE',
      label: 'Folgen Sie auf Facebook'
    },
    {
      name: 'twitter',
      url: 'https://twitter.com/smartfrog_cam',
      label: 'Auf Twitter folgen'
    },
    {
      name: 'youtube',
      url: 'https://www.youtube.com/channel/UCt2tBYCxnfhBWraIuOcY2Ow',
      label: 'Folgen Sie auf YouTube'
    },
    {
      name: 'instagram',
      url: 'http://instagram.com/smartfrogcam/',
      label: 'Folgen Sie auf Instagram'
    }
  ]
}

describe('Social Icon component', () => {
  it('Should render component and match snapshot', () => {
    const wrapper = mount(SocialIcon, { propsData: socialItems })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
