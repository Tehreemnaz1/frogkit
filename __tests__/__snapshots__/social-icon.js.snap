// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social Icon component Should render component and match snapshot 1`] = `<a href="http://www.facebook.com/smartfrogDE" target="_blank" rel="noopener" class="fk-social-icon facebook">Folgen Sie auf Facebook</a>`;
